roles/cobbler_profile/defaults/main.yml

```diff
@@ -0,0 +1,22 @@
+---
+# Where to download ISOs
+iso_dir: /var/lib/cobbler/isos
+# Mount point to use for ISOs during import
+iso_mount: /mnt/iso
+
+distros:
```

Comment:
An actual distribution artifact (ISO) is the input that creates a distro/profile pair. When we reimage, we are acting on system and profile objects; not distros. 

So it's weird, but exactly as weird as the above statement :)